components/esp_lvgl_port/esp_lvgl_port.c

```diff
@@ -23,12 +23,6 @@
 #include "esp_lcd_touch.h"
 #endif
 
-#if ESP_IDF_VERSION < ESP_IDF_VERSION_VAL(5, 0, 1)
```

Comment:
You still need to check if IDF is 4.4.4 or earlier (e.g. v4.4.3), don't you?

Reply:
Yes, fixed.